### DISCUSSION TITLE
Guide: security-oidc-bearer-token-authentication-tutorial

### DISCUSSION THREAD

**Discussion by trym-moeller** [quarkusio/quarkus#37971] [Community]
Hi,

I am following the guide: https://quarkus.io/guides/security-oidc-bearer-token-authentication-tutorial,
And think that the step 1. Create the Maven Project is missing an extension parameter in the command. Something like -Dextensions='oidc'?

Best regards Trym

**Comment by sberyozkin** [comment]
@trym-moeller Thanks, fixing now, the macro is there but looks like some extra doc style characters affect its expansion

**Reply by sberyozkin** [reply]
@trym-moeller FYI, #37975

**Reply by trym-moeller** [reply]
Thanks for the quick response. I enjoy the guides, so just wanted them even better :-)

**Reply by sberyozkin** [reply]
Thanks @trym-moeller
